docs/authentication/authentication.html

```diff
@@ -89,12 +89,6 @@ <h1 class="docs-title"><a href="/index.html">KITURA <span class="blue-text">DOCS
       <div class="burger-line"></div>
       <div class="burger-line"></div>
     </div>
-    <div class="docs-item-2 search-container">
```

Comment:
You don't need to remove this entire container doing so breaks the styling. 

You should only need to remove the following: 
```html
<div id="beta" class="temp-beta">	
        <h4>Currently in Beta - <a href="../../old-learn.html">View old docs</a></h4>	
</div>
```

Same will apply to every other instance.